PowerEditor/installer/themes/Solarized.xml

```diff
@@ -779,15 +1157,6 @@ Installation:
             <WordsStyle name="SPECIAL" styleID="11" fgColor="CB4B16" bgColor="002B36" fontName="" fontStyle="0" fontSize="" />
             <WordsStyle name="COMMENT" styleID="12" fgColor="586E75" bgColor="002B36" fontName="" fontStyle="2" fontSize="" />
         </LexerType>
-        <LexerType name="searchResult" desc="Search result" ext="">

```

Comment:
Was this done intentionally?

Reply:
This is now on [line 1379](https://github.com/notepad-plus-plus/notepad-plus-plus/blob/71bff9c877911b190af690426251c8d52228c122/PowerEditor/installer/themes/Solarized.xml#L1379). I thought to move it so that it matches `stylers.xml`, and hence makes future edits easier for people.

I can put it back in its original place if you would like me to do so.